### PR TITLE
Split test tree to have unit and integration tests

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: packages/yourloops/test/integration/utils/auth0.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+export const mockAuth0 = () => {
+  (auth0Mock.useAuth0 as jest.Mock).mockReturnValue({
+    isAuthenticated: true,
+    isLoading: false,
+    user: {
+      email: 'john.doe@example.com',
+      email_verified: true,
+      sub: 'auth0|fakeUserId',
+      'http://your-loops.com/roles': ['hcp']
```
Comment (fatb38) [inline packages/yourloops/test/integration/utils/auth0.ts:38]: You can use the enum `UserMetadata` in models/user.ts
```suggestion
      [UserMetadata.Roles]: ['hcp']
```

**Comment by fatb38** [comment]
What do you think to regroup the utils files related to the UI (in integration tests) into a single file `UI-components` or something like this

**Review by fatb38 — APPROVED** [review]

**Review by fatb38 — CHANGES_REQUESTED** [review]
I saw remaining useless mock like in `main-drawer.test` with children property. These mocks could be removed !

**Review by Nas84 — CHANGES_REQUESTED** [review]
Seems that the jest config is duplicated, several solution exist to remove the duplication:
- leverage projects parameters
- shared a configuration module and import it

reference: [https://jestjs.io/docs/configuration#projects-arraystring--projectconfig](https://jestjs.io/docs/configuration#projects-arraystring--projectconfig)

**Review by Nas84 — APPROVED** [review]
LGTM

**Pull request merged**